extensions/amp-subscriptions/amp-subscriptions.md

```diff
@@ -348,6 +348,36 @@ For instance, to show a "subscribe" action to non-subscribers:
 <button subscriptions-display="NOT subscribed" subscriptions-action="subscribe">Become a subscriber</button>
 ```
 
+## Analytics
+
+The `amp-subscriptions` triggers seven types of analytics signals during the different lifecycle of various subscriptions.
+Following are the events and the conditions when the events are triggered.
+
+1. PLATFORM_ACTIVATED:
```

Comment:
Use the actual values that can be used in the analytics. E.g. "subscriptions-platform-activated".

Reply:
Done